components/datatypes/mobile-application.json

```diff
@@ -0,0 +1,133 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/mobile-application",
```

Comment:
We use the "https://ns.adobe.com/xdm/datatypes/......" for all new data types we define now.